modules/smart-agent_zookeeper/conf/00-zookeeper-health.yaml

```diff
@@ -0,0 +1,16 @@
+module: zookeeper
```

Comment:
This detector should aggregate on all servers in the cluster and trigger a major on loss of part of the servers (half ? third ?) and critical on loss of more than that.

Reply:
For example : 
```
signal = data('gauge.zk_service_health', filter=filter('env', 'preprod') and filter('sfx_monitored', 'true')).mean(by=['plugin_instance']).publish('signal')
detect(when(signal < 0.66, lasting='5m', at_least=1)).publish('CRIT')
detect(when(signal < 1, lasting='5m', at_least=1)).publish('MAJ')```